### DISCUSSION TITLE
I am getting funny values for the gradient class bg-gradient

### DISCUSSION THREAD

**Discussion by keef-ui** [twbs/bootstrap#31260] [General] [answered]
When i compile the scss file to css i get following odd values for bg-gradient in the css :

 .bg-gradient {
  background-image: var(--bs-gradient) !important; 
}

note : I have enabled the following in variables : $enable-gradients:            true !default;

**Comment by mdo** [comment, marked as answer]
`var(--bs-gradient)` is the syntax for CSS custom properties, aka CSS variables. See https://v5.getbootstrap.com/docs/5.0/customize/css-variables/ for some details.

**Reply by keef-ui** [reply]
Thanks for that it makes sense...I guess this is not widely supported in most browsers yet.

**Reply by patrickhlauke** [reply]
https://caniuse.com/#feat=css-variables
